modules/highgui/include/opencv2/highgui.hpp

```diff
@@ -125,6 +125,10 @@ It provides easy interface to:
         cv::createTrackbar("Twin brother", windowName, &state, 100, callbackTwin);
     }
     @endcode
+
+    @note In modern usage, it is recommended to pass `nullptr` for the value pointer if you do not need
+    to update it directly. Instead, handle the trackbar position manually in the callback function.
+    See the updated trackbar tutorial for more details    
```

Comment:
take a look at test results
https://pullrequest.opencv.org/#/summary/opencv
https://pullrequest.opencv.org/buildbot/builders/precommit_docs/builds/109520

please remove extra space in this line

Reply:
Also, what is "handle manually" supposed to mean? Please explain there, so people aren't required to read and understand the example.